bookwyrm/templates/guided_tour/home.html

```diff
@@ -99,7 +99,7 @@
         ],
     },
     {
-        text: "{% trans 'Use the <strong>Feed</strong>, <strong>Lists</strong> and <strong>Discover</strong> links to discover the latest news from your feed, lists of books by topic, and the latest happenings on this Bookwyrm server!' %}",
+        text: "{% trans 'Use the <strong>Feed</strong>, <strong>Discover</strong>, and <strong>Your Books</strong> links to discover the latest news from your feed, the latest happenings on this Bookwyrm server, and see your catalogued books!' %}",
```

Comment:
This should be Lists, Discover, and Your Books.

Reply:
Actually it probably should be worded a bit differently to, something like:

_...to discover reading suggestions and the latest happenings on this server, or to see your catalogued books_